tests/src/deviceLib/hipSimpleAtomicsTest.cpp

```diff
@@ -242,6 +242,7 @@ int main(int argc, char** argv) {
     hipDeviceReset();
     printf("%s completed, returned %s\n", sampleName, testResult ? "OK" : "ERROR!");
     exit(testResult ? EXIT_SUCCESS : EXIT_FAILURE);
+    passed();
```

Comment:
This doesn't look right. The previous statement is exit(). So this piece of code is going to be unreachable.